### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AlexisEidelman**
I think the only conflict you have is the new sum in for "equipement". I've imported that change, so you can force the merging if you validate the change.

Note than if you accept it, Makefile can be removed and th doc has to be changed a little.

### CODE REVIEW

**Comment by armgilles** [comment]
@AlexisEidelman 

i PR #29 new data (census 2012). Could you update your PR ?

File: insee/equipements.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    assert len(df) == df.CODGEO.nunique()
+    nb_features = len(df.columns) - len(columns_not_to_sum)
+    print("\t il y a ", df.CODGEO.nunique(), " iris différentes pour et ",
+          nb_features, " features")
+    return df
+
+
+def info_equipement(year):
+    equipement = None
+    for table in equipements:
```
Comment (armgilles) [inline insee/equipements.py:93]: what is equipements ?

File: insee/equipements.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+#                                'NB_C503', 'NB_C504', 'NB_C509',
+#                                'NB_C601', 'NB_C602', 'NB_C603',
+#                                'NB_C604', 'NB_C605', 'NB_C609',
+#                                'NB_C701', 'NB_C702']),
+#     social = ('equip-serv-action-sociale', None),
+#     sante = ('equip-serv-sante', None),
+#     medical = ('equip-serv-medical-para', None),
+#     service_particulier = ('equip-serv-particuliers', None),
+#     transport_tourisme = ('equip-tour-transp', None),
+     )
```
Comment (armgilles) [inline insee/equipements.py]: Missing #

File: insee/main.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+from globals import path_data
+
+print("Initialisation...")
+from equipements import info_equipement
+from revenus import info_revenus
+from census import info_population
+
+
+equipement = info_equipement()
```
Comment (armgilles) [inline insee/main.py]: Missing arguement no ?

info_equipement(2011)

File: insee/equipements.py [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    assert name in equipements
+    filename, list_to_sum = equipements[name]
+    df = read_equipement_file(filename, year)
+    # a bug in one excel... 'equip-serv-commerce'
+    df.dropna(how='any', axis=1, inplace=True)
+    assert all(df.isnull().sum() == 0)
+#    df = sum_of_all_features(df, 'nb_' + name, list_to_sum)
+    assert len(df) == df.CODGEO.nunique()
+    nb_features = len(df.columns) - len(columns_not_to_sum)
+    print("\t il y a ", df.CODGEO.nunique(), " iris différentes pour et ",
```
Comment (armgilles) [inline insee/equipements.py:85]: I have this kind of message : 
" ('\t il y a ', 50867, ' iris diff\xc3\xa9rentes pour et ', 77, ' features') "

File: insee/equipements.py (hunk 1 above)
Comment (AlexisEidelman) [inline insee/equipements.py:93]: Sorry, I'll fix that. 
equipements is the list of all equipements tables

File: insee/equipements.py (hunk 2 above)
Comment (AlexisEidelman) [inline insee/equipements.py]: Sorry again, what I pull requested was kind of dirty...
My apologises for that.

File: insee/main.py (hunk 3 above)
Comment (AlexisEidelman) [inline insee/main.py]: yes.
Sor far 2013 and 2014 are implemented. I didn't check what the link was for years before 2013 on the insee website

File: insee/equipements.py (hunk 4 above)
Comment (AlexisEidelman) [inline insee/equipements.py:85]: I think it's a python 2/ python 3 conflict.
I've inserted 
`from __future__ import print_function`
see http://python-future.org/compatible_idioms.html

Comment (armgilles) [inline insee/equipements.py:85]: :+1:

**Comment by AlexisEidelman** [comment]
Seems that just using `info_population(2012)` does the work for 2012 automaticaly.
Still I don't know how to reuse what you've done about merging year 2011 and 2012

**Comment by armgilles** [comment]
I just add features. 

I'll try to test your PR ASAP.

File: insee/equipements.py [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+    return iris_df
+
+
+#key = ['CODGEO', 'LIBGEO', 'COM', 'LIBCOM', 'REG', 'DEP', 'UU2010']
+
+columns_not_to_sum = ['CODGEO', 'LIBGEO', 'COM', 'LIBCOM', 'REG', 'DEP', 'ARR', 'CV',
+             'ZE2010', 'UU2010', 'UU12010', 'ID_MODIF_GEO']
+
+# Useless
+#equipements = dict(
```
Comment (armgilles) [inline insee/equipements.py]: I uncomment equipements for info_equipement()

**Comment by armgilles** [comment]
I think you can easily add census 2012. There are 2 tricks with [new region code (2016)](https://github.com/anthill/open-moulinette/blob/master/insee/mk_data.py#L379) and [fix_LIBGEO_12](https://github.com/anthill/open-moulinette/blob/master/insee/mk_data.py#L408)

When it's done, i will test if there is no difference between original output.csv and the new one.

File: insee/main.py [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+
+data = data.merge(population, how='outer')
+## petit bout de code pour voir ce qui s'ajoute
+#cond = equip_data.CODGEO.isin(revenu.CODGEO)
+#equip_data[cond]
+#cond = revenu.CODGEO.isin(equip_data.CODGEO)
+#revenu[~cond]
+
+data.to_csv('output.csv')
```
Comment (armgilles) [inline insee/main.py]: should be 'data/output.csv'

**Pull request closed without merging**